soc/silabs/silabs_siwx91x/Kconfig

```diff
@@ -14,3 +14,21 @@ config SOC_SILABS_SLEEPTIMER
 	bool
 	help
 	  The Sleeptimer HAL module is used for SIWX91X.
+
+if SOC_FAMILY_SILABS_SIWX91X
+config SIWX91X_SILABS_COMMANDER_SIGN
+	bool "Silicon Labs Commander Signing Support"
```

Comment:
In fact, the use can still launch `west sign` if this option is not set. Probably we want to fix the description of this option.